stablehlo/dialect/StablehloOps.td

```diff
@@ -2448,9 +2449,6 @@ def StableHLO_ReduceWindowOp: StableHLO_Op<"reduce_window", [
     See https://www.tensorflow.org/xla/operation_semantics#reducewindow.
   }];
 
-  // TODO(hinsu): Verify that padding attribute is 2-d and the remaining
-  // attributes are 1-d. Attributes' leading dimension should match rank of the
-  // operands.
```

Comment:
It looks like `convertPaddingAttribute` still accepts 1D tensors. Is that intended?

Reply:
nope, I can't find any evidence that 1D padding should be supported in StableHLO, so I'm removing it now. It may lead to some error when merging this back to MHLO (maybe break some tests there), but will keep in mind of this to fix/remove tests there too.

Reply:
Sounds good. Let's see if we can get away with this.